src/sentry/static/sentry/app/views/organizationMonitors/monitors.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import {Link, withRouter} from 'react-router';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import {Box} from 'grid-emotion';
+
+import AsyncView from 'app/views/asyncView';
+import BetaTag from 'app/components/betaTag';
+import {Panel, PanelBody, PanelItem} from 'app/components/panels';
+import {PageHeader} from 'app/styles/organization';
+import PageHeading from 'app/components/pageHeading';
+import TimeSince from 'app/components/timeSince';
+import Pagination from 'app/components/pagination';
+import SentryTypes from 'app/sentryTypes';
+import SearchBar from 'app/components/searchBar';
+import withOrganization from 'app/utils/withOrganization';
+import {t} from 'app/locale';
+
+import MonitorIcon from './monitorIcon';
+
+const HeaderTitle = styled(PageHeading)`
+  flex: 1;
+`;
+
+const StyledSearchBar = styled(SearchBar)`
+  flex: 1;
+`;
+
+class OrganizationMonitors extends AsyncView {
+  static contextTypes = {
```

Comment:
Should this be propTypes instead of contextTypes? You don't read from context and the wrapper components add props.